docs/pipeline.rst

```diff
@@ -37,10 +37,8 @@ unit tests, as well as the smoke suite of redirect headless HTTP(s) response che
 Push to master branch
 ~~~~~~~~~~~~~~~~~~~~~
 
-Whenever a change is pushed to the master branch, the smoke suite of
-headless (see :ref:`testing-redirects`) and UI tests (see :ref:`smoke-functional-tests`)
```

Comment:
Hmm, should we still run the redirect tests at this step? I'm not clear if those will get run against a deployment or not, looking at the current pipeline?

Reply:
Could we move the redirect tests to run against a deployment instead perhaps?

Reply:
I was under the impression that we were already running the redirect tests against the deployments as part of the headless test suite, but I'm double checking now.

Reply:
@alexgibson we are running the redirect tests against remote, but we don't currently appear to run tests with the smoke marker against remote deployments. I've pushed a new commit that should run them against the deployments in parallel to the other tests, and that's running in https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/296/pipeline/ now

Reply:
@jgmize at this point I really wonder what use the smoke markers are doing. Why run those only in Firefox in a separate run, and not just run all the tests for each browser?

Reply:
Also, I do see the smoke tests running against the other browser runs. Happy to chat about this more in person if it helps.

Reply:
Good points @alexgibson, I agree the smoke markers aren't adding any value anymore, and I've pushed a new commit which eliminates them entirely and all references to them.